schema/common/defaults/configuration/rook.yml

```diff
@@ -0,0 +1,6 @@
+---
+kind: configuration/rook
+title: "Kubernetes Rook Config"
+name: default
+specification:
```

Comment:
I don't see specification.cluster_chart_values and specification.operator_chart_values.

Reply:
I will add optional validation, but won't specify anything here.